conf/php-fpm.ini

```diff
@@ -0,0 +1,10 @@
+# Common values to change to increase file upload limit
```

Comment:
Here and on L6, we need to convert "#" to ";" as otherwise we get this message every 30 minutes:
```
PHP Deprecated:  Comments starting with '#' are deprecated in /etc/php5/fpm/conf.d/20-ttrss.ini on line 1 in Unknown on line 0
```
We need to modify `example_ynh/confphp-fpm.ini` accordingly.

Here should we fix the file or simply delete it...?

Reply:
You can remove it, it's a default file.  
On my side, I will update the file in the example app !